src/Workspaces/Core/Portable/Storage/ISQLiteStorageServiceFactory.cs

```diff
@@ -7,10 +7,10 @@
 namespace Microsoft.CodeAnalysis.Storage
 {
     /// <summary>
-    /// Factory for SQLite storage service - intentionally not included under SQLite directory since all sources under it are excluded from source build.
+    /// Provider for SQLite storage service - intentionally not included under SQLite directory since all sources under it are excluded from source build.
     /// </summary>
-    internal interface ISQLiteStorageServiceFactory : IWorkspaceService
+    internal interface ISQLiteStorageServiceProvider : IWorkspaceService
```

Comment:
Rename file to `ISQLiteStorageServiceProvider`

Reply:
done.